smartsim/settings/arguments/launchArguments.py

```diff
@@ -37,25 +37,32 @@
 logger = get_logger(__name__)
 
 
-class LaunchArgBuilder(ABC):
+class LaunchArguments(ABC):
     """Abstract base class that defines all generic launcher
     argument methods that are not supported.  It is the
```

Comment:
"argument methods that are not supported. " -> "argument methods that are by default not implemented. "